tutorials/13_diagnostics_simulation_based_calibration.ipynb

```diff
@@ -541,13 +984,78 @@
+   "id": "c2b021c4-c1aa-433d-95c4-e09e926f172f",
    "metadata": {},
    "source": [
     "The above provides a visual representation of the cumulative density function (CDF) of `ranks` (blue and orange for each dimension of `theta`) with respect to the 95% confidence interval of a uniform distribution (grey).\n"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "b3f1a0cc-d83f-4f0c-b174-ca367aab7699",
+   "metadata": {},
+   "source": [
+    "## multi dimensional SBC\n",
+    "\n",
+    "So far, we have performed the SBC checks for each dimension of our parameters $\\theta$ separately. SBI offers a way to perform this check for all dimensions at once."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c4de299-54a2-4ebd-beb0-344e90767c9f",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "c56e48670ecb47309f927ff55fd6df2b",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Running 1000 sbc samples.:   0%|          | 0/1000 [00:00<?, ?it/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'ks_pvals': tensor([0.1461]), 'c2st_ranks': tensor([0.5555]), 'c2st_dap': tensor([0.5100, 0.4925])}\n"
+     ]
+    }
+   ],
+   "source": [
+    "# use reduce_fns in order to map the test statistic to only one dimension\n",
+    "ranks, dap_samples = run_sbc(\n",
+    "    thetas,\n",
+    "    xs,\n",
+    "    posterior,\n",
+    "    num_posterior_samples=num_posterior_samples,\n",
+    "    reduce_fns=posterior.log_prob,\n",
+    ")\n",
+    "check_stats = check_sbc(ranks, thetas, dap_samples, 1_000)\n",
+    "print(check_stats)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a283036-93e2-44ff-ac1d-fe42ceaaf6a0",
+   "metadata": {},
+   "source": [
+    "In the code above, we depart from the default behavior of `run_sbc`. The standard behavior of `run_sbc` is to calculate the SBC ranks of parameters $\\theta$ by comparing the marginal values of the predicted parameter $\\theta_i$ to the reference value $\\theta_o$, i.e. we are ranking each parameter only within its marginal dimension `idx` which evaluates `theta_i[idx] < theta_o[idx]` to perform sbc ranking.  \n",
+    "\n",
+    "The `reduce_fns` parameter of `run_sbc` allows users to specify a `Callable` which is invoked on the tuple `(theta,x)` before the sbc ranking is performed. As an example, we specify `reduce_fns=posterior.log_prob` here to rank according to the log probability of theta (given x) under the posterior. This results in the ranking being performed in one dimension (the log probabilities) instead of each marginal dimension of theta separately. Internally, `run_sbc` would then evaluate `posterior.log_prob(theta_i, x_i) < posterior.log_prob(theta_o, x_o)` to perform the ranking.\n",
```

Comment:
sorry to be persistent here: It's much better now, except one sentence: 
```suggestion
    "The `reduce_fns` parameter of `run_sbc` allows users to specify a `Callable` which is invoked on the tuple `(theta,x)` before the sbc ranking is performed. As an example, we specify `reduce_fns=posterior.log_prob` here to rank according to the log probability of theta (given x) under the posterior. This results in the ranking being performed across all dimensions of `theta` at once (instead of each marginal dimension separately) by *reducing* `theta` to the log probabilities under the posterior. Internally, `run_sbc` would then evaluate `posterior.log_prob(theta_i, x_o) < posterior.log_prob(theta_o, x_o)` to perform the ranking.\n",
```